src/view/screens/LanguageSettings.tsx

```diff
@@ -48,6 +48,7 @@ export function LanguageSettingsScreen(_props: Props) {
 
   const onChangePrimaryLanguage = React.useCallback(
     (value: Parameters<PickerSelectProps['onValueChange']>[0]) => {
+      if (!value) return
```

Comment:
In case the placeholder ever shows back up, it has a value of `null`.

Reply:
Cool